src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/data/MessageContexts.kt

```diff
@@ -14,3 +19,9 @@ class MigrationContext<T>(val type: MigrationType, val migrationId: String, val
 }
 
 fun generateBatchId(): String = LocalDateTime.now().withNano(0).format(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
+
```

Comment:
First use is when sentencing synchronisation posts a message to itself to retry mapping, we no longer have to add dummy "migration" attributes